src/test/scala/com.snowplowanalytics.iglu.client/validation/SelfDescValidationSpec.scala

```diff
@@ -48,58 +49,62 @@ class SelfDescValidationSpec extends Specification with ValidationMatchers {
     """{"schema": "iglu:com.snowplowanalytics.iglu-test/stock-item/jsonschema/1-0-0", "data": { "id": "123-12", "name": "t-shirt", "price": 29.99 } }"""
   )
 
-  def e1 = validJson.validate(false) must beSuccessful(validJson)
+  def e1 = validJson.validate(false) must beValid(validJson)
 
-  def e2 = validJson.validate(true) must beSuccessful(validJson.get("data"))
+  def e2 = validJson.validate(true) must beValid(validJson.get("data"))
 
   val invalidJson = SpecHelpers.asJsonNode(
     """{"schema": "iglu:com.snowplowanalytics.iglu-test/stock-item/jsonschema/1-0-0", "data": { "id": "123-12", "newName": "t-shirt", "price": "tbc" } }"""
   )
-  val invalidExpected = NonEmptyList(
-    SpecHelpers.asProcessingMessage(
-      message =
-        """object instance has properties which are not allowed by the schema: ["newName"]""",
-      schema = """{"loadingURI":"#","pointer":""}""",
-      instance = """{"pointer":""}""",
-      keyword = "additionalProperties",
-      foundExpected = None,
-      requiredMissing = None,
-      unwanted = Some("""["newName"]""")
-    ),
-    SpecHelpers.asProcessingMessage(
-      message = """object has missing required properties (["name"])""",
-      schema = """{"loadingURI":"#","pointer":""}""",
-      instance = """{"pointer":""}""",
-      keyword = "required",
-      foundExpected = None,
-      requiredMissing = Some("""["id","name","price"]""", """["name"]"""),
-      unwanted = None
+  val invalidExpected = NonEmptyList
+    .of(
+      SpecHelpers.asProcessingMessage(
+        message =
+          """object instance has properties which are not allowed by the schema: ["newName"]""",
+        schema = """{"loadingURI":"#","pointer":""}""",
+        instance = """{"pointer":""}""",
+        keyword = "additionalProperties",
+        foundExpected = None,
+        requiredMissing = None,
+        unwanted = Some("""["newName"]""")
+      ),
+      SpecHelpers.asProcessingMessage(
+        message = """object has missing required properties (["name"])""",
+        schema = """{"loadingURI":"#","pointer":""}""",
+        instance = """{"pointer":""}""",
+        keyword = "required",
+        foundExpected = None,
+        requiredMissing = Some("""["id","name","price"]""", """["name"]"""),
+        unwanted = None
+      )
     )
-  ).map(_.toString)
+    .map(_.toString)
 
-  def e3 = invalidJson.validate(false).leftMap(_.map(_.toString)) must beFailing(invalidExpected)
+  def e3 =
+    invalidJson
+      .validate(false)
+      .leftMap(_.map(_.toString)) must beInvalid // TODO: check expected messages
```

Comment:
?

Reply:
Just a reminder to test the format after it's decided